content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx

```diff
@@ -0,0 +1,249 @@
+---
+title: 'Indexes'
+metaDescription: 'How to configure index functionality and add full text indexes'
+hidePage: false
+tocDepth: 2
+preview: true
+---
+
+<TopBlock>
+
+Prisma allows configuration of database indexes, unique constraints and primary key constraints through the `extendedIndexes` preview feature in version `3.5.0` and later.This provides extended syntax options in the Prisma schema language for configuring the length of indexes (MySQL only), the sort order and the index type (PostgreSQL only).
```

Comment:
```suggestion
Prisma allows configuration of database indexes, unique constraints and primary key constraints through the `extendedIndexes` preview feature in version `3.5.0` and later.This provides extended syntax options in the Prisma schema language for configuring the length of indexes (MySQL only), the sort order (all databases) and the index type (PostgreSQL only).
```
Is this suggested edit correct @pimeys?

Reply:
I've added this in for now as I'm pretty sure it's correct.